ftl/python/builder.py

```diff
@@ -132,9 +137,24 @@ def _whl_to_fslayer(self, whl):
         tmp_dir = tempfile.mkdtemp()
         pkg_dir = os.path.join(tmp_dir, 'env')
         os.makedirs(pkg_dir)
-        subprocess.check_call(
-            ['pip', 'install', '--prefix', pkg_dir, whl],
-            env=self._gen_pip_env())
+
+        pip_cmd_args = list(self._pip_cmd)
+        pip_cmd_args.extend(['install', '--no-deps', '--prefix', pkg_dir, whl])
+
+        proc_pipe = subprocess.Popen(
+            pip_cmd_args,
+            stdin=subprocess.PIPE,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            env=self._gen_pip_env(),
+        )
+        stdoutdata, stderrdata = proc_pipe.communicate()
```

Comment:
just stdout and stdin?

Reply:
i have changed this to just stdout and stderror